bin/console

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env php
+<?php
+
+declare(strict_types=1);
+
+use Libero\DummyApi\Kernel;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Debug\Debug;
+
+set_time_limit(0);
+
+require __DIR__.'/../vendor/autoload.php';
+
+$input = new ArgvInput();
+
+$env = $_SERVER['APP_ENV'] ?? 'dev';
```

Comment:
was discussing recently (on Twitter) how explicit names for environments prevent reuse in novel environments; especially in an open source application where we cannot hardcode environment names inside the code. The aspect we are trying to model here is whether to turn on debug or not, therefore an effective environment variable could be a boolean `APP_DEBUG`

Reply:
will likely wreak havoc on `config/packages/`, but it seems that there `dev` and `test` are replicating configuration that could explicitly chosen for a random $environment if needed (e.g. turn on debug or the profiler in `staging` temporarily)

Reply:
Funnily enough I took out the [default `APP_DEBUG`](https://github.com/symfony/recipes/blob/5b3ce909504b9366405820c49bc9f0e13ac4be54/symfony/framework-bundle/4.2/public/index.php#L19) in favour of a bit of simplification. Will restore.

I think we could go for a compromise, eg eLife's `continuumtest` and `end2end` are `prod` + a few settings. As long as there are environment variables for those things (eg disable email sending) that will work.

Reply:
It's a concept with lower cardinality, mostly `dev`|`test` (but are they actually different and where?) vs `prod` and its replicas. Would be a good to have an explicit name for it that isn't overloaded on `environment`.